src/main/scala/uk/gov/nationalarchives/draftmetadatavalidator/S3Files.scala

```diff
@@ -5,26 +5,26 @@ import cats.implicits.catsSyntaxOptionId
 import org.typelevel.log4cats.SelfAwareStructuredLogger
 import uk.gov.nationalarchives.aws.utils.s3.S3Utils
 import uk.gov.nationalarchives.draftmetadatavalidator.ApplicationConfig.fileName
-import uk.gov.nationalarchives.draftmetadatavalidator.Lambda.{DraftMetadata, getFilePath, getFolderPath}
+import uk.gov.nationalarchives.draftmetadatavalidator.Lambda.{ValidationParameters, getFilePath, getFolderPath}
 
 import java.io.File
 import java.nio.file.Paths
 import scala.reflect.io.Directory
 
 class S3Files(s3Utils: S3Utils)(implicit val logger: SelfAwareStructuredLogger[IO]) {
 
-  def key(draftMetadata: DraftMetadata) = s"${draftMetadata.consignmentId}/$fileName"
+  def key(validationParameters: ValidationParameters) = s"${validationParameters.consignmentId}/$fileName"
 
-  def downloadFile(bucket: String, draftMetadata: DraftMetadata): IO[Any] = {
+  def downloadFile(bucket: String, validationParameters: ValidationParameters): IO[Any] = {
 
-    cleanup(getFolderPath(draftMetadata))
-    val filePath = getFilePath(draftMetadata)
+    cleanup(getFolderPath(validationParameters))
+    val filePath = getFilePath(validationParameters)
     if (new File(filePath).exists()) {
       logger.info("")
```

Comment:
Do we need this `logger.info("")` if we don't have any messages?

Reply:
definitely not